packages/devtools_app/test/provider/provider_controller_test.dart

```diff
@@ -2,27 +2,29 @@
 // Use of this source code is governed by a BSD-style license that can be
 // found in the LICENSE file.
 
-// ignore_for_file: import_of_legacy_library_into_null_safe
-
+// ignore: import_of_legacy_library_into_null_safe
+import 'package:devtools_app/devtools_app.dart';
```

Comment:
we should import libraries directly instead of importing devtools_app.dart, which exports many libraries.

Reply:
Good call, done